src/AppBundle/Entity/Report/Report.php

```diff
@@ -467,6 +478,7 @@ public function __construct(Client $client, $type, \DateTime $startDate, \DateTi
             $statusCached[$sectionId] = ['state' => ReportStatusService::STATE_NOT_STARTED, 'nOfRecords' => 0];
         }
         $this->setStatusCached($statusCached);
+        $this->statusStatusCached = self::STATUS_NOT_STARTED;
```

Comment:
typo? statusStatusCached

Reply:
yes. fixed and pushed. PHPStorm didn't notice, weird